Jenkinsfile

```diff
@@ -9,7 +9,6 @@ node {
         sh 'git clean -fxd'
     }
     stage('test') {
-        sh 'python2.7 -V'
```

Comment:
can't be py3_only yet. vsc-utils is still in use on centos 7 machines.

Reply:
Can we version lock it there?

Reply:
as long as it is in use on python2.7 I can not agree to making it py3 only.